src/lib/stitching/gravity/v2/stitching.ts

```diff
@@ -205,17 +210,51 @@ const gravityMarketingCollectionsResolvers = (gravitySchema) => {
         },
       },
     },
+    MarketingCollection: {
+      artworksConnection: {
+        fragment: `
+          ... on MarketingCollection {
+            internalID
+          }
+        `,
+        resolve: (
+          { internalID: marketingCollectionID },
+          args,
+          context,
+          info
+        ) => {
+          return info.mergeInfo.delegateToSchema({
+            schema: localSchema,
+            operation: "query",
+            fieldName: "artworksConnection",
+            args: {
+              marketingCollectionID,
+              ...(!!context.currentArtworkID && {
+                excludeArtworkIDs: [context.currentArtworkID],
+              }),
```

Comment:
@nickskalkin was this added so that in the future if we implement an "other works in this collection" rail, we don't show the work the user is already viewing (similar to what already exists for artist series)? 

If so, I would consider leaving it out for now because it's hard to know if/when we would actually implement that rail. Not something I feel extremely strong about, but I'm generally in favor of avoiding the addition of code that we're not 100% sure we need 🙂

Reply:
good catch, thanks! Removed